### PR TITLE
Scale homepage modal buttons

### LINKED ISSUE #1174: Homepage modal buttons not scaled
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### :fire: Problem

Screenshot_20191225_171442

Screenshot_20191225_171519

On mobile phone and tablets, buttons are not of the same height.
It results in bad UI/UX and it should be fixed.

### 🚩 Solution

Screenshot_20191225_171912

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Open** the file ./mapknitter/app/views/front_ui/index.erb

- Line 55 -> add inline styles to paragraph
```html
<p style="display:flex;justify-content:space-around;"><i style="color:#aaa;">A Community Atlas hosted by <a href="https://publiclab.org">PublicLab</a></i></p>
```

- Line 64 -> add an inline style to the anchor tag
```html
<a class="btn btn-primary btn-lg" style="display:flex;align-items:center;justify-content:center;" href="" data-toggle="collapse" data-target="#anonymous-map">
```

- Line 66 -> add margin-left to inline styles
```html
<i class="fa fa-plus fa-fw" style="font-size:12px;color:white;margin-left:2%;"></i>
```

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
$DIFF
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by sakshi-2412**
Fixes #1174 (<=== Add issue number here)

Scaled homepage modal buttons so that they are of the same height on all screens

https://user-images.githubusercontent.com/85152262/156917991-067a1173-6371-46e2-9c47-139cbb035e7a.mp4

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

[//]: # (To mark checkbox write 'x' within the square brackets)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/mapknitter-reviewers` for help, in a comment below

### CODE REVIEW

File: app/views/front_ui/index.html.erb [hunk 1]
```diff
@@ -16,18 +16,18 @@
...
             <p><i style="color:#aaa;">A Community Atlas hosted by <a href="https://publiclab.org">PublicLab</a></i></p>
             <p>Use a kite, balloon, or pole to take an aerial photo and tell your own visual story of a place. <a href="/about" class="alert-link">Learn more</a></p>
-            <p>
+            <div class="modal-buttons">
             <a class="btn btn-primary btn-lg" href= 'https://store.publiclab.org/collections/mapping-kits'>
                 Get a mapping kit <i class="fa fa-angle-double-right fa-fw" style="font-size:18px;color:white;"></i>
             </a>
             <% if logged_in? %>
-              <a class="btn btn-primary btn-lg" href="<%= new_map_url %>">
+              <a class="btn btn-primary btn-lg" href="<%= new_map_url %>" style="display:flex; align-content:center; flex-wrap:wrap; justify-content:center;">
```
Comment (VladimirMikulic) [inline app/views/front_ui/index.html.erb:24]: You can refactor these duplicated inline styles into `.modal-buttons a` selector.

**Review by VladimirMikulic — CHANGES_REQUESTED** [review]

File: app/views/front_ui/index.html.erb (hunk 1 above)
Comment (sakshi-2412) [reply app/views/front_ui/index.html.erb:24]: Sure! I'll do that

**Comment by sakshi-2412** [comment]
@VladimirMikulic I have updated the changes.. can you please take a look. Thanks

**Review by VladimirMikulic — APPROVED** [review]

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
I'm trying to fix some unrelated dependency issues in https://github.com/publiclab/mapknitter/pull/1616 which should unblock this -- we'll need to rebase again, but hang on and let me try to get this working!

**Comment by jywarren** [comment]
https://github.com/publiclab/mapknitter/pull/1616 succeeded!!!! Can you rebase this over the latest main branch? I believe it will then pass. Thank you!!!

**Comment by sakshi-2412** [comment]
Did I do something wrong? I had to resolve some merge conflicts, not sure if I did it right :disappointed:

**Comment by jywarren** [comment]
Hi, I'm so sorry to reply late. I'm going to try rebasing this over the latest main branch and I /think/ it will resolve -- nothing you did wrong, in any case!

**Comment by jywarren** [comment]
OK, rebased! Let's see - 🤞

**Comment by jywarren** [comment]
OK phew, sorry that took a while, it's merged now! Thank you!!

**Pull request merged**